### ISSUE TITLE
Add method for "apply_service"

### ISSUE THREAD

**Issue by krihal** [clicon/clixon-pyapi#17]
apply_service in the Clixon class should be added with the possibility to do a diff.

**Comment by krihal** [comment]
Fixed with e8649ac6d4d0ac303eeb44f21c2b6e81635d9be4.

**Comment by krihal** [comment]
Example:

```
from clixon.clixon import Clixon

clixon = Clixon()

data = clixon.apply_service("ssh-users", "test", diff=True)

print(data)
```
